### ISSUE TITLE
Whitespace throws warning during coversion

### ISSUE THREAD

**Issue by cudencuden** [philip/MySQLConverterTool#24]
```
$result = SQL_Query_exec("SELECT SUM(downloaded) AS totaldl FROM users"); 
while ($row = mysql_fetch_array ($result)) { 
	$totaldownloaded = $row["totaldl"]; 
} 
```
**edit: Solved, it is just the space between mysql_fetch_array and ($result)**

I noticed that for every pattern of SUM above i get: Please check your code for parse errors, we failed to parse " ". Conversion will be incomplete!". The warning referenced line is at WHILE, but I am thinking the SUM() is throwing the error. any help?

**Comment by philip** [comment]
Thanks, I'll check next week, but wonder if it might relate to mixing user defined and native functions like that. Do you see the same error when using mysql_query()?

**Comment by cudencuden** [comment]
you are correct, i do not see same warning when using mysql_query(). the user defined class escapes the $query argument; there is no warning for user defined class except when SUM is used as original message

**Comment by philip** [comment]
Interesting. I'm only able to do a little testing now, but also found a way to make your code work. Removing the space from `mysql_fetch_array (` somehow converts fine, so this works:
```
$result = SQL_Query_exec("SELECT SUM(downloaded) AS totaldl FROM users"); 
while ($row = mysql_fetch_array($result)) { 
	$totaldownloaded = $row["totaldl"]; 
}
```
I'm not yet sure why this is the case as whitespace shouldn't interfere here. Perhaps while testing mysql_query() you also happened to remove that space? Or maybe there are multiple issues going on here. Ugh.

Also, it appears the conversion works fine despite this bogus warning, so at least there's that :)

**Comment by cudencuden** [comment]
yes, it is just the space between mysql_fetch_array and ( . thank  you.

**Comment by philip** [comment]
This shouldn't be a problem, let's reopen this.
